packages/@aws-cdk/aws-stepfunctions-tasks/lib/ecs/run-task.ts

```diff
@@ -318,7 +327,7 @@ export class EcsRunTask extends sfn.TaskStateBase implements ec2.IConnectable {
     const policyStatements = [
       new iam.PolicyStatement({
         actions: ['ecs:RunTask'],
-        resources: [this.props.taskDefinition.taskDefinitionArn],
+        resources: [this.props.useTaskDefinitionFamily ? '*' : this.props.taskDefinition.taskDefinitionArn],
```

Comment:
link to any documentation here that indicates that further resource scoping is not possible and it must be scoped to `*`

I thought the full ARN included a version and that was the only part that needed to be dropped.

Reply:
Yes I checked it by deploying and executing the machine. Dropping the revision works!